manager/templates/default/css/index.css

```diff
@@ -23889,3 +23889,8 @@ iframe[classname="x-hidden"] {
 
 .x-datetime-inline-editor .x-datetime-wrap {
   margin-top: 0 !important; }
+
+/*mark required fields*/
+.field-required-mark {
+  color: red;
```

Comment:
Too expressive I guess, bold will be enough.

Reply:
fields labels already are bold. I removed red style for 'required asterix'